controllers/homework.js

```diff
@@ -642,6 +654,9 @@ router.get('/:assignmentId/edit', function (req, res, next) {
 		assignment.availableDate = moment(assignment.availableDate).format('DD.MM.YYYY HH:mm');
 		assignment.dueDate = moment(assignment.dueDate).format('DD.MM.YYYY HH:mm');
 
+		addClearNameForFileIds(assignment);
```

Comment:
In addClearNameForFileIds heißt der parameter noch submission_s, sollte man umbenennen.

Reply:
überarbeitet